docs/medical-api/fhir/overview.mdx

```diff
@@ -0,0 +1,134 @@
+
+A FHIR Resource is a representation of some concept related to healthcare. Some examples include:
+
+- [Encounter](/medical-api/fhir/resources/encounter): an interaction between a patient and healthcare provider(s).
+- [MedicationRequst](/medical-api/fhir/resources/medicationrequest): an order or request for both supply of the medication and the instructions for administration of the medication to a patient.
+- [Observation](/medical-api/fhir/resources/observation): measurements about a patient - ie weight, height, blood pressure, etc.
+
+In JSON, an Observation Resource would look like this, for example:
+
+```json
+{
+  "resourceType": "Observation",
+  "id": "D19DA918-0F16-11EE-B2A6-A2F55214B7C1",
+  "meta": {
+    "versionId": "1",
+    "lastUpdated": "2023-06-19T15:37:06.127+00:00",
+    "source": "https://api.metriport.com/cda/to/fhir#flkjasdfoiueh"
+  },
+  "extension": [
+    {
+      "url": "https://public.metriport.com/fhirextensions#observation-documentDate",
+      "valueDateTime": "2019-02-22T16:24:00+00:00"
+    },
+    {
+      "url": "https://public.metriport.com/fhirextensions#observation-documentType",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "https://public.metriport.com/codes/Proprietary/DocumentType",
+            "code": "CCD Vitals",
+            "display": "CCD Vitals",
+            "userSelected": true
+          }
+        ]
+      }
+    },
+    {
+      "url": "https://public.metriport.com/fhir/StructureDefinition/doc-id-extension.json",
+      "valueString": "1A5512D2-0EE9-11EE-B2A6-A2F55214B7C1-MLKJ392874kjshhKJSHDFdskjfh98234uiy"
+    }
+  ],
+  "status": "unknown",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "vital-signs",
+          "userSelected": false
+        }
+      ],
+      "text": "vital-signs"
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "urn:oid:2.16.840.1.113883.6.1",
+        "code": "8302-2",
+        "display": "Body height",
+        "userSelected": true
+      },
+      {
+        "system": "http://loinc.org",
+        "code": "8302-2",
+        "display": "Body height",
+        "userSelected": false
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/2.16.840.1.113883.3.9621.666"
+  },
+  "encounter": {
+    "reference": "Encounter/D19DA9AE-0F16-11EE-B2A6-A2F55214B7C1"
+  },
+  "effectiveDateTime": "2019-02-22T16:24:00+00:00",
+  "issued": "2019-02-22T16:24:00+00:00",
+  "valueQuantity": {
+    "value": 167.6,
+    "unit": "centimeter",
+    "system": "http://unitsofmeasure.org",
+    "code": "cm"
+  }
+}
+```
+
+You're able to query for a Patient's FHIR Resources using the [Get Patient's Consolidated Data
+Endpoint](/medical-api/api-reference/fhir/get-patient-consolidated).
+
+Additionally, to meet data contribution requirements, you're able to provide new data about a
```

Comment:
Wondering if we could have a link on `data contribution requirements` to the description about that? It's good to have product people involved on our requirements, they might not be aware of requirements since they're not the ones signing contracts, in most cases.

Reply:
Will reword this to avoid overhead & another link - FYI this requirement is in the ToS

Reply:
Makes sense - and we should probably describe this in more detail in general: ticket here https://github.com/metriport/metriport/issues/597